src/layout/InsertionForm.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import { Form, Col, Row, Button, InputGroup } from 'react-bootstrap';
+
+function InsertionForm(props) {
```

Comment:
instead of passing props it is better destructure props here, like so `function InsertionForm({ addExpense, addIncome })`. This will pass linter check here.